### PR TITLE
agenda filters feature flag

### PR DESCRIPTION
**Pull request by SidelineCory24**
Note: filters are UI only at this time

### CODE REVIEW

File: public/config/config.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "post_handshake": false,
     "post": true,
     "register_checklist": true,
-    "agenda_search": true
+    "agenda_search": true,
+    "agenda_filters": false
```
Comment (elizabeth-jimenez) [inline public/config/config.json:34]: you can set it to true since we already pushed along to the code fro 8.0

**Comment by SidelineCory24** [comment]
the test that are failing on the same test on the original PR

**Review by elizabeth-jimenez — APPROVED** [review]
👍

**Comment by SidelineCory24** [comment]
@mjoyce91 is there a way to override the `exit status 137` error to allow the pr to merge?

**Comment by mjoyce91** [comment]
@SidelineCory24 - you should be good now

**Pull request merged**